.travis.yml

```diff
@@ -62,6 +67,19 @@ deploy:
       repo: SkygearIO/skygear-server
       branch: master
       go: 1.13.3
+  - provider: s3
+    access_key_id: "$AWS_ACCESS_KEY"
+    secret_access_key: "$AWS_SECRET_KEY"
```

Comment:
About the setup,
1. Would like to know is those key ready in travis?
2. When I visit `code.skygear.dev`, seems it is not ready yet. Open another issue for completing the setup?

Reply:
> 1

Yes

> 2

Will set up this today.